.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## What changes were made
+
+<!-- Describe the changes you did -->
+
+## How to test or check results
+
+<!-- URL or instructions -->
+
+## Checks
+- [ ] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
+- [ ] Appointed PR reviewers. At least 1 [code owner](./codeowners) should review
```

Comment:
So it seems that GH does this automatically right?

Reply:
Yes you're right. Let's remove it!